src/types.rs

```diff
@@ -21,7 +21,7 @@ pub type IOPhysicalLength64  = u64;
 
 
 #[cfg(target_pointer_width = "64")] pub type IOVirtualAddress = mach_vm_address_t;
-#[cfg(target_pointer_width = "32")] pub type IOVirtualAddress = vm_address_t;
+#[cfg(target_pointer_width = "32")] pub type IOVirtualAddress = mach_vm_address_t;
```

Comment:
This doesn't match the header file that I see in XCode:

```c
#if !defined(__arm__) && !defined(__i386__)
typedef mach_vm_address_t   IOVirtualAddress
#else
typedef vm_address_t        IOVirtualAddress
#endif
```

Reply:
Yes, right, I opened fitzgen/mach#12 for adding the missing `vm_address_t` type